redis/src/commands/mod.rs

```diff
@@ -1257,6 +1263,157 @@ implement_commands! {
         cmd("ZUNIONSTORE").arg(dstkey).arg(keys.num_of_args()).arg(keys).arg("AGGREGATE").arg("MAX").arg("WEIGHTS").arg(weights)
     }
 
+    // vector set commands
+
+    /// Add a new element into the vector set specified by key.
+    /// [Redis Docs](https://redis.io/commands/VADD)
+    #[cfg(feature = "vector-sets")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "vector-sets")))]
+    fn vadd<K: ToRedisArgs, E: ToRedisArgs>(key: K, input: VectorAddInput<'a>, element: E) -> (bool) {
+        cmd("VADD").arg(key).arg(input).arg(element)
+    }
+
+    /// Add a new element into the vector set specified by key with optional parameters for fine-tuning the insertion process.
+    /// [Redis Docs](https://redis.io/commands/VADD)
+    #[cfg(feature = "vector-sets")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "vector-sets")))]
+    fn vadd_extended<K: ToRedisArgs, E: ToRedisArgs>(key: K, input: VectorAddInput<'a>, element: E, options: &'a VAddOptions) -> (bool) {
```

Comment:
please be consistent with the naming scheme - we use `_options`, not `_extended`.

Reply:
Thank you for pointing this out.